Code/Tools/FBuild/FBuildCore/Graph/VSProjectBaseNode.cpp

```diff
@@ -0,0 +1,53 @@
+// VSProjectBaseNode.cpp
+//------------------------------------------------------------------------------
+
+// Includes
+//------------------------------------------------------------------------------
+#include "VSProjectBaseNode.h"
+
+// Reflection
+//------------------------------------------------------------------------------
+
+REFLECT_NODE_BEGIN( VSProjectBaseNode, Node, MetaName("ProjectOutput") + MetaFile() )
+REFLECT_END( VSProjectBaseNode )
+
+
+// CONSTRUCTOR
+//------------------------------------------------------------------------------
+VSProjectBaseNode::VSProjectBaseNode()
+    : FileNode( AString::GetEmpty(), Node::FLAG_ALWAYS_BUILD )
+{
+    m_Type = Node::PROXY_NODE;
+    m_LastBuildTimeMs = 100; // higher default than a file node
+}
+
+
+// Initialize
+//------------------------------------------------------------------------------
+/*virtual*/ bool VSProjectBaseNode::Initialize(NodeGraph& /*nodeGraph*/, const BFFIterator& /*iter*/, const Function* /*function*/)
+{
+    // Generate default platform / config tuples if not specified
```

Comment:
The external project will have these configurations by default, which have no relation to what's in the project file, so these defaults being in this base class don't seem to make sense.

I imagine the ideal outcome would be for the External node to parse and extract the projects present in the file?